app/controllers/authorisationsAndLimit/authorisations/index/AuthorisationReferenceNumberController.scala

```diff
@@ -70,7 +70,7 @@ class AuthorisationReferenceNumberController @Inject() (
             }
 
             Ok(view(preparedForm, lrn, request.arg.forDisplay, mode, authorisationIndex, approvedOperator))
-          case _ => Redirect(config.sessionExpiredUrl)
+          case _ => Redirect(config.sessionExpiredUrl(lrn))
```

Comment:
Consider redirecting to tech difficulties here and logging that the approved operator value could not be determined

Reply:
Updated now.